[inductor] raise tolerance for cspdarknet

shunting314 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #127949

cspdarknet previously is flaky but after https://github.com/pytorch/pytorch/pull/127367 it fails quite stably. It's probably due to small numerical change from the mentioned PR. That PR will let inductor generated different code due to different loop orders.

Raise tolerance to pass CI.




cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @kadeng @chauhang

shunting314 committed:
[inductor] raise tolerance for cspdarknet

[ghstack-poisoned]

nWEIdia reviewed (approved):
LGTM! Let's make sure to check the timm shards to verify this is indeed helping.

nWEIdia commented:
It looks like  cudnn v9 update PR ([merged](https://github.com/pytorch/pytorch/commit/c490046693e77e254664e19d940e9b05a1da18ef)) also brought the model back to pass? 

cc @eqy if the above is true, we may not need to raise the tolerance.

eqy commented:
I'm fine with merging this as I would not bet on cuDNN heuristics remaining unchanged if e.g., this update changed them

nWEIdia commented:
For the period (like now) that cuda 12.4 has not been brought to parity with cuda 12.1, could you please also update the benchmarks/dynamo/ci_expected_accuracy/cu124/dynamic_inductor_timm_training.csv file to change cspdarknent from Fail_Accuracy to pass?

shunting314 committed:
Update on "[inductor] raise tolerance for cspdarknet"


cspdarknet previously is flaky but after https://github.com/pytorch/pytorch/pull/127367 it fails quite stably. It's probably due to small numerical change from the mentioned PR. That PR will let inductor generated different code due to different loop orders.

Raise tolerance to pass CI.




cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 kadeng chauhang

[ghstack-poisoned]

nWEIdia commented:
@pytorchbot merge -f "the changes in this PR only affects cspdarknet and previous commit of it is showing passing."
